File: trunk/D2Packets/BnetClientPacket.cs
using System;

namespace D2Packets
{
	public enum BnetClientPacket : byte
	{
        KeepAlive               = 0x00,
        LeaveChat               = 0x10,
        AdInfoRequest           = 0x15,
        StartGame               = 0x1C,
		LeaveGame			    = 0x1F,
        DisplayAd               = 0x21,
        NotifyJoin              = 0x22,
        BnetPong                = 0x25,
        FileTimeRequest         = 0x33,
        BnetLogonRequest        = 0x3A,
        RealmLogonRequest       = 0x3E,
        QueryRealms             = 0x40,
        NewsInfoRequest         = 0x46,
        ExtraWorkResponse       = 0x4B,
        BnetConnectionRequest   = 0x50,
        BnetAuthRequest         = 0x51,
		Invalid				    = 0xA0,	// Probably 0x83, leaving some space for safety...
        EnterChatRequest        = 0x0A,
        ChannelListRequest      = 0x0B,
        JoinChannel             = 0x0C,
        ChatCommand             = 0x0E
    
    }
}

File: D2PacketsVB/D2PacketsVB/Bnet/Client/BnetClientPacket.vb
Namespace BnetClient
    Public Enum BnetClientPacket

        KeepAlive = 0
        EnterChatRequest = 10
        ChannelListRequest = 11
        JoinChannel = 12
        ChatCommand = 14
        LeaveChat = 16
        AdInfoRequest = 21
        StartGame = 28
        LeaveGame = 31
        DisplayAd = 33
        NotifyJoin = 34
        BnetPong = 37
        FileTimeRequest = 51
        BnetLogonRequest = 58
        RealmLogonRequest = 62
        QueryRealms = 64
        NewsInfoRequest = 70
        ExtraWorkResponse = 75
        BnetConnectionRequest = 80
        BnetAuthRequest = 81

        Invalid = 131

    End Enum
End Namespace
